scripts/benchmarks-ci.sh

```diff
@@ -48,17 +45,23 @@ for pallet in ${pallets[@]}
 do
 	output_dir=""
 	extra_args="--template=$benchmarkTemplates/runtime-weight-template.hbs"
+	extrinsic="--extrinsic=*"
 	# A little hack for pallet_xcm_benchmarks - we want to force custom implementation for XcmWeightInfo.
 	if [[ "$pallet" == "pallet_xcm_benchmarks::generic" ]] || [[ "$pallet" == "pallet_xcm_benchmarks::fungible" ]]; then
 		output_dir="xcm/"
 		extra_args="--template=$benchmarkTemplates/xcm-bench-template.hbs"
 	fi
+	if [[ "$pallet" == "pallet_xcm_benchmarks::generic" ]]; then
+	  # Exclude extrinsics that involved features not present in pop atm.
+	  # A custom configuration for runtime-benchmarks feature could be provided in the runtime for the relevant types otherwise.
+	  extrinsic="--extrinsic=report_holding,unpaid_execution,set_fees_mode,clear_topic,set_topic,clear_transact_status,report_transact_status,expect_pallet,query_pallet,expect_transact_status,expect_error,expect_origin,expect_asset,burn_asset,unsubscribe_version,subscribe_version,trap,claim_asset,report_error,clear_origin,execute_with_origin,descend_origin,clear_error,set_appendix,set_error_handler,refund_surplus,transact,query_response,asset_claimer,pay_fees,buy_execution,report_holding"
+  fi
 	$artifactsDir/pop-node benchmark pallet \
 		$extra_args \
 		--chain=$specName \
```

Comment:
Had a version using `--runtime`, but for some reason wasn't working properly, so came back to chain. Not specially happy keeping this in, but it does work.

Hopefully `pop` will be a nicer solution.